### ISSUE TITLE
ZooKeeper: logLevel parameter doesn`t work

### ISSUE THREAD

**Issue by everythings-gonna-be-alright** [bitnami/charts#11440]
### Name and Version

bitnami/zookeeper 10.0.1

### What steps will reproduce the bug?

```
- name: ZOO_LOG_LEVEL
  value: ERROR
```

### What do you see instead?

```
2022-07-31 00:43:08,922 [myid:] - INFO  [QuorumPeer[myid=3](plain=0.0.0.0:2181)(secure=disabled):o.a.z.s.q.QuorumPeer@1892] - Dynamic reconfig is disabled, we don't store the last seen config.
2022-07-31 00:43:08,928 [myid:] - INFO  [QuorumPeer[myid=3](plain=0.0.0.0:2181)(secure=disabled):o.a.z.s.q.QuorumPeer@922] - Peer state changed: following - synchronization
2022-07-31 00:43:08,932 [myid:] - INFO  [QuorumPeer[myid=3](plain=0.0.0.0:2181)(secure=disabled):o.a.z.s.q.CommitProcessor@491] - Configuring CommitProcessor with readBatchSize -1 commitBatchSize 1
2022-07-31 00:43:08,932 [myid:] - INFO  [QuorumPeer[myid=3](plain=0.0.0.0:2181)(secure=disabled):o.a.z.s.q.CommitProcessor@452] - Configuring CommitProcessor with 1 worker threads.
```

**Comment by aoterolorenzo** [comment]
Marked as on-hold to avoid stale bot from closing the issue meanwhile working on it

**Comment by aoterolorenzo** [comment]
Hi @everythings-gonna-be-alright,

Thanks for taking the time to file the issue. I can say that it is already solved since chart 10.1.0 version release.

**Comment by Pio-kaw** [comment]
@aoterolorenzo 
Thank you for the update. Do you know whether it only occurs on 10.0.1? I'm experiencing the same issue on 8.1.1
